config/locales/api/si_LK.yml

```diff
@@ -89,14 +89,14 @@ si_LK:
     bp_passport_app_summary:
       new_html: අලුත්!
       heading_html: ඔබගේ රෝගීන් සඳහා යෙදවුම
-      contents_1_html: "\"බීපී පාස්පෝට්\" නම් සරල යෙදවුමකින්, දැන් රෝගීන්ට රුපී ය සහ රුධිරයේ සීනි මට්ටම වගු ගත කළ හැකිය."
+      contents_1_html: "\"බීපී පාස්පෝට්\" නම් simple යෙදවුමකින්, දැන් රෝගීන්ට රුපී ය සහ රුධිරයේ සීනි මට්ටම වගු ගත කළ හැකිය."
```

Comment:
This is translation for

> Patients can now chart their own BPs and blood sugars with a simple app called the "BP Passport" app.

"simple" should not be in English, since it's not referring to our software.

I will let Vineet know in #localization